### PR TITLE
Update swearwords.go

### PR DESCRIPTION
**Pull request by ROWDYGAMER5**
Added a couple of common bypasses to a couple of words (N-words and F-slur).

### CODE REVIEW

**Comment by Wolveric** [comment]
_Direct committing to master branch **REEEEEE**_, plus, just... the base list isn't really getting updated at this point. Just create your own on a server by server basis...

**Comment by mrbentarikau** [comment]
don't PR to master

**Comment by jo3-l** [comment]
From `CONTRIBUTING.md`:

> If you want to make a contribution, create a fork, then push the changes you make to the dev branch (Jonas747/yagpdb/dev). This is key for larger commits and edits to important / complicated parts of the bot so it can be tested safely without breaking everything.

**Comment by l-zeuch** [comment]
from [CONTRIBUTING.md](./CONTRIBUTING.md):
> push the changes you make to the dev branch

File: automod_legacy/swearwords.go [hunk 1]
```diff
@@ -46,6 +49,8 @@ var BuiltinSwearWords = map[string]bool{
 	"muff":        true,
 	"nigger":      true,
 	"nigga":       true,
+	"n1gga":       true,
+	"n1gger":       true,
```
Comment (jo3-l) [inline automod_legacy/swearwords.go:53]:
```suggestion
	"n1gger":      true,
```
In any case, if you do wish to resubmit this to the proper branch, you will want to fix this (`gofmt` is making the CI fail currently).

**Review by jo3-l — CHANGES_REQUESTED** [review]

**Pull request closed without merging**